src/Grids/regular_cartesian_grid.jl

```diff
@@ -53,13 +53,15 @@ Make sure to specify the desired `FT` if not using `Float64`.
 
 Grid properties
 ===============
+
 - `(xC, yC, zC)::AbstractRange`: (x, y, z) coordinates of cell centers
 - `(xF, yF, zF)::AbstractRange`: (x, y, z) coordinates of cell faces
 - `(Hx, Hy, Hz)::Int`: Halo size in the (x, y, z)-direction
 - `(Tx, Ty, Tz)::Int`: "Total" grid size (interior + halo points) in the (x, y, z)-direction
```

Comment:
```suggestion
```

We should remove this line as `Tx, Ty, Tz` are no longer grid properties. Probably my fault for not removing this in the PR where I made this change.